iocore/net/SSLUtils.cc

```diff
@@ -1686,7 +1685,7 @@ SSLInitServerContext(const SSLConfigParams *params, const ssl_user_config *sslMu
       }
 
       for (const char *certname = cert_tok.getNext(); certname; certname = cert_tok.getNext()) {
-        completeServerCertPath = Layout::relative_to(params->serverCertPathOnly, certname);
+        ats_scoped_str completeServerCertPath(Layout::relative_to(params->serverCertPathOnly, certname));
```

Comment:
Declare this as `std:string` - that would avoid an unnecessary string duplication.